plugins/wysiwygarea/plugin.js

```diff
@@ -515,15 +525,9 @@
 			detach: function() {
 				var editor = this.editor,
 					doc = editor.document,
-					iframe,
+					iframe = editor.container.findOne( 'iframe' ),
```

Comment:
Is there no edge case where the container may be already detached? I'm also not sure about your changes. I have a feeling like getting `editor.window.getFrame` is safer than searching container (eg in some weird case where editor's container have additional iframe element added by 3rd party plugins like context menu).

Can't we just check iframe by `iframe && iframe.isDetached`?

Reply:
Also, please, cover these changes by unit tests.

Reply:
`editor.window.getFrame();` returns `null` when iframe is detached, but this doesn't guarantee that custom data has been cleaned, so we have potential memory leaks.

I can improve selector.
```suggestion
					iframe = editor.container.findOne( 'iframe.cke_wysiwyg_frame' ),
```